### DISCUSSION TITLE
ACE - Arsenal

### DISCUSSION THREAD

**Discussion by Philipp-00** [KillahPotatoes/KP-Liberation#972] [Q&A]
Is it possible to configure Liberation in such a way that I can add an ACE Arsenal to the mission, which provides selected material, specifically a "whitelist", to the players in the mission?

**Comment by KiwiTwitches** [comment]
This is possible through the Custom arsenal section in the presets folder. presets/arsenal/custom.sqf

An example of one I have for a CUP AFRF arsenal. 
[custom.txt](https://github.com/user-attachments/files/23137435/custom.txt)
